index.js

```diff
@@ -4,27 +4,51 @@ const arrify = require('arrify');
 const isDirectory = require('is-directory');
 const pify = require('pify');
 
-const getGlob = (fp, ext) => path.join(fp, '**', ext || '');
-const getExt = ext => Array.isArray(ext) ? `*.{${ext.join(',')}}` : `*.${ext}`;
-const getPath = fp => fp[0] === '!' ? fp.slice(1) : fp;
+const isDirectoryP = pify(isDirectory);
```

Comment:
Instead of using `pify` here, just use https://github.com/sindresorhus/path-type instead